C/Coin-OR/coin-or-common.jl

```diff
@@ -64,7 +64,7 @@ ALPS_gitsha = "5b1a0b524979764d6ca929446762762712c035bb"
 ALPS_version_offset = v"0.0.2"
 ALPS_version = offset_version(ALPS_upstream_version, ALPS_version_offset)
 
-BiCePS_version = v"0.9.4"
+BiCePS_version = offset_version(v"0.94.4", v"0.0.0")
```

Comment:
Is `0.94.4` a typo?

Reply:
No, the previous version was a typo: https://github.com/coin-or/CHiPPS-BiCePS/releases/tag/releases%2F0.94.4

The naming scheme of the releases leaves something to be desired:

![image](https://user-images.githubusercontent.com/8177701/144729672-d4616d84-93a2-42ba-a586-92a3db212c22.png)